CMakeLists.txt

```diff
@@ -332,6 +332,15 @@ if(CMAKE_USE_OPENSSL)
   endif()
 endif()
 
+option(USE_NGHTTP2 "Use Nghttp2 library" OFF)
+if(USE_NGHTTP2)
+  find_package(NGHTTP2 REQUIRED)
+  if(NGHTTP2_FOUND)
```

Comment:
Note that this is strictly not necessary since you have REQUIRED for find_package.